tcms/rpc/tests/test_bug.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+from mock import patch, MagicMock
+
+from django.conf import settings
+
+from tcms.rpc.tests.utils import APITestCase
+
+if 'tcms.bugs.apps.AppConfig' not in settings.INSTALLED_APPS:
+    raise unittest.SkipTest('tcms.bugs is disabled')
+
+
+class TestBug(APITestCase):
+    def _fixture_setup(self):
+        super()._fixture_setup()
+        self.url = "http://some.url"
+        self.expected_result = {
+            'title': 'Bug from cache',
+            'description': 'This bug came from the Django cache'
+        }
+
+    @patch('tcms.rpc.api.bug.tracker_from_url')
+    @unittest.skip('debug')
+    def test_get_details_from_tracker(self, tracker_from_url):
+        returned_tracker = MagicMock()
+        returned_tracker.details.return_value = self.expected_result
+        tracker_from_url.return_value = returned_tracker
```

Comment:
The mock chain here looks good. Strictly speaking you don't need the mocks here, could use a full blown setup of all the components then the test becomes an integration type test.  The existing mocking in this case introduces a slight speedup and helps not deal with the dependencies like creating a new bug + new bug tracker, etc.

IMO can stay like that and should pass from what I can see.